lib/main.jsx

```diff
@@ -6,9 +6,11 @@ var TEST_DATA = require('./test-data.json');
 
 var parser = new GraphParser();
 var graph = parser.parseGraph(TEST_DATA);
+var SpringLayoutEngine = require('./layout/forceDirectedLayout');
 
 // rerender on any change
 // graph.after('change', renderGraph);
+SpringLayoutEngine(graph);
```

Comment:
A function with a name starting with an uppercase letter should only be used as a constructor.